content/recipes/findability/findingCompounds.md

```diff
@@ -0,0 +1,80 @@
+# Finding FAIR Experimental Datasets for Chemical Compounds
+ "Using Wikidata to find experimental data related chemical compounds in a FAIR dataset."
+
+
+ ````{panels_fairplus}
+:identifier_text: FCB053 
```

Comment:
See comment by @proccaserra here: https://github.com/FAIRplus/the-fair-cookbook/pull/353